multimedia/spotify_dl/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                spotify_dl
+version             7.6.0
+revision            0
+
+categories          multimedia python
+platforms           darwin
+supported_archs     noarch
+license             GPL-2
+maintainers         {judaew @judaew} openmaintainer
+
+description         Downloads songs from any Spotify playlist, album or track.
+long_description    {*}${description}
+homepage            https://github.com/SathyaBhat/spotify-dl
+
+checksums           rmd160  b4306d38b60ee39aaccf570d3761da7ab928f9f7 \
+                    sha256  158c08bc5e3bcccfea8c46bfa19eef880b5fbb57c31ac375ed332ef80c5410a2 \
+                    size    9714
+
+post-extract {
+    copy ${filespath}/requirements.txt ${worksrcpath}
+}
```

Comment:
can you please report this to upstream that they should include it in their PyPI files?

Reply:
Done, https://github.com/SathyaBhat/spotify-dl/issues/225